MoltenKT-Paper/src/main/resources/plugin.yml

```diff
@@ -10,4 +10,8 @@ softdepend:
   - HeadDatabase
   - PlaceholderAPI
 prefix: ${name}
-website: ${website}
\ No newline at end of file
+website: ${website}
+loadbefore:
+  - SimpleCloud-Plugin
+depend:
+  - Kotlin
```

Comment:
Kotlin Libraries are already included in the runnable, why depend on the use of an external Kotlin-Plugin?
Maybe this was not intended?

Reply:
For a good reason, it needs to have the kotlin lib depend so if it try's to load kotlin, it would ether load kotlin from itself or from the other kotlin plugin euch provides the same version. 
You could also say it's a try and catch for an exception.

Reply:
But now it forces the external Kotlin plug-in to be additionally installed on your server?

Reply:
I'll try to run it without the external kotlin in the evening, then I can for sure state the answer correctly.